esbuild.dev.config.ts

```diff
@@ -16,16 +15,16 @@ esbuild
       entryPoints: ["./src/index.tsx"],
       outfile: "public/script.js",
       sourcemap: true,
-      external: [...Object.keys(dependencies), 'react/jsx-runtime'],
+      external: ['react/jsx-runtime'],
       plugins: [
+        svgrPlugin({ exportType: 'named' }),
         inlineImage(),
         sassPlugin({
           async transform(source) {
-            const { css } = await postcss([autoprefixer]).process(source);
+            const { css } = await postcss([autoprefixer]).process(source, { from: undefined });
```

Comment:
```suggestion
            const {css} = await postcss([autoprefixer]).process(source, {from: undefined});
```
We don't add padding in curly braces, right?.....fix it everywhere

Reply:
wondering why the linter didn't give error for this? @tanisha-agarwal